sbnanaobj/StandardRecord/SREnums.h

```diff
@@ -44,7 +44,9 @@ namespace caf
   enum generator_
   {
     kUnknownGenerator = 0,
-    kGENIE            = 1
+    kGENIE            = 1,
+    kMeVPrtlHiggs     = 2,
```

Comment:
These new enum flags are a bit misleading. 
GENIE is a generator, MevPrtHiggs and MeVPrtlHNL are physics channels. 
I would preferred something like MevPrtl which is the big umbrella that contains all this work. Or even simpler, `kSBN` or `kSBNCode` because that is where the input is coming from. 
I do realise that it means we need to change part of the SBN code.

Reply:
Fixed! And didn't need a change to sbncode actually (although yes to sbana somehow).